src/app/components/HoldRequest/HoldRequest.jsx

```diff
@@ -136,6 +118,22 @@ class HoldRequest extends React.Component {
   }
 
   /**
+   * modelDeliveryLocationName(prefLabel, shortName)
+   * Renders the names of the radio input fields of delivery locations except EDD.
+   *
+   * @param {String} prefLabel
+   * @param {String} shortName
+   * @return {String}
+   */
+  modelDeliveryLocationName(prefLabel, shortName) {
+    if (prefLabel && typeof prefLabel === 'string' && shortName) {
+      return `${shortName} - ${prefLabel.split(' - ')[1]}`;
```

Comment:
Yeah, I'd suggest you do the changes all over again in the correct branch instead of branching and merging from dev-eb-deploy. You can also accidentally add changes that haven't been approved yet or are still being worked on (it is a deployment branch).